metasmoke.py

```diff
@@ -224,7 +310,7 @@ def send_feedback_for_post(post_link, feedback_type, user_name, user_id, chat_ho
         if GlobalVars.metasmoke_host is None:
```

Comment:
Note for the not too distant future:

In GlobalVars, we really should have something like:

    GlobalVars.metasmoke_host_valid = GlobalVars.metasmoke_host and GlobalVars.metasmoke_key
    GlobalVars.metasmoke_ws_host_valid = GlobalVars.metasmoke_ws_host and GlobalVars.metasmoke_key

and then use `GlobalVars.metasmoke_host_valid` and `GlobalVars.metasmoke_ws_host_valid` in almost all the various places where we're testing for it being possible to use MS or the MS WebSocket. As it is, we inconsistently test throughout various parts of the code for a Truthy/Falsy value, against `None`, or `!= ""` and for only one of the required pieces of data or the other.

Reply:
Hopefully we can get all metasmoke_* variables refactored into a class MSConfig and implement is_host_valid() and is_ws_host_valid().